src/pharmpy/tools/iivsearch/algorithms.py

```diff
@@ -105,7 +105,7 @@ def _is_current_block_structure(etas, combos):
 def _iiv_param_dict(model: Model) -> Dict[str, str]:
     iiv = model.random_variables.iiv
     return {
-        eta: get_rv_parameter(model, eta)
+        eta: get_rv_parameters(model, eta)[0]
```

Comment:
Maybe it would make sense to have a convenience `_get_rv_parameter` that raises an intelligible error if the number of located parameters is not 1. That could even be a user-facing function. What do you think @rikardn?

Reply:
Yeah, there should maybe be some check here, or even earlier in input change. However, not sure it need to raise, it is only for description so if a user has the same eta on multiple parameters, the actual tool will still work as expected, it's just the descriptions that will not match with what's happening.

Reply:
Also I will be rewriting this dict in the PR where I update the description of iovsearch, so maybe this is okay as is for this PR?

Reply:
Yep.